commit 405e52f412b85b581899f5e1b82d25a7c8959d89
Author: Pierre-Louis Bossart <pierre-louis.bossart@linux.dev>
Date:   Fri Nov 19 17:13:27 2021 -0600

    ASoC: SOF: sof-pci-dev: use community key on all Up boards
    
    There are already 3 versions of the Up boards with support for the SOF
    community key (ApolloLake, WhiskyLake, TigerLake). Rather than
    continue to add quirks for each version, let's add a wildcard.
    
    For WHL and TGL, the authentication supports both the SOF community
    key and the firmware signed with the Intel production key. Given two
    choices, the community key is the preferred option to allow developers
    to sign their own firmware. The firmware signed with production key
    can still be selected if needed with a kernel module
    option (snd-sof-pci.fw_path="intel/sof")
    
    Tested-by: Péter Ujfalusi <peter.ujfalusi@linux.intel.com>
    Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
    Reviewed-by: Bard Liao <yung-chuan.liao@linux.intel.com>
    Reviewed-by: Péter Ujfalusi <peter.ujfalusi@linux.intel.com>
    Link: https://lore.kernel.org/r/20211119231327.211946-1-pierre-louis.bossart@linux.intel.com
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/sof/sof-pci-dev.c b/sound/soc/sof/sof-pci-dev.c
index bc9e70765678..b4bc4f887b43 100644
--- a/sound/soc/sof/sof-pci-dev.c
+++ b/sound/soc/sof/sof-pci-dev.c
@@ -64,17 +64,9 @@ static const struct dmi_system_id sof_tplg_table[] = {
 
 static const struct dmi_system_id community_key_platforms[] = {
 	{
-		.ident = "Up Squared",
+		.ident = "Up boards",
 		.matches = {
 			DMI_MATCH(DMI_SYS_VENDOR, "AAEON"),
-			DMI_MATCH(DMI_BOARD_NAME, "UP-APL01"),
-		}
-	},
-	{
-		.ident = "Up Extreme",
-		.matches = {
-			DMI_MATCH(DMI_SYS_VENDOR, "AAEON"),
-			DMI_MATCH(DMI_BOARD_NAME, "UP-WHL01"),
 		}
 	},
 	{